app/client/views/apply/applyCtrl.js

```diff
@@ -257,7 +269,7 @@ angular.module('reg')
               rules: [
                 {
                   type: 'empty',
-                  prompt: 'Please select your major.'
+                  prompt: 'Please type your major.'
```

Comment:
Let’s revert this change since you’re continuing to use a drop down menu for major selection